src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -2472,4 +2472,41 @@
     </Environment>
   </Provider>
 
+  <!--
+                                                ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
```

Comment:
You used the wrong font name and forgot the trailing space: https://documentation.openiddict.com/guides/contributing-a-new-web-provider#add-the-ascii-art-header

Reply:
I think I used the correct one `Small Negative`. But noticed that we should use all caps "which is not part of the manual".